tools/release/Publish-BondNuGetPackages.ps1

```diff
@@ -0,0 +1,116 @@
+<#
+    .SYNOPSIS
+        Publishes Bond NuGet packages to NuGet feeds.
+
+    .PARAMETER PackageSource
+        The directory with the packages to publish.
+
+    .PARAMETER PackageFeed
+        The feed to publish to.
+
+    .PARAMETER SkipSymbols
+        Whether to skip pushing ".symbols.nupkg" packages and just push
+        ".nupkg" pakages. Defaults to false
+
+    .PARAMETER NuGetPath
+        The full path to NuGet.exe to use. If not specificed, attempts to use
+        NuGet.exe from $env:PATH.
+
+    .PARAMETER ApiKey
+        The API Key to use to authenticate to the feed.
+        An ApiKey is not needed for VSTS feeds, as these need to use a
+        credential provider. One is needed for NuGet.org, however.
+#>
+
+[CmdletBinding(SupportsShouldProcess=$True)]
+param
+(
+    [Parameter(Mandatory=$True)]
+    [string]
+    $PackageSource,
+    [Parameter(Mandatory=$True)]
+    [string]
+    $PackageFeed,
+    [switch]
+    $SkipSymbols = $false,
```

Comment:
should this be false on default?

Reply:
Let me rename this to OmitSymbols

Yes: being false will include symbols. That's what we want.
PowerShell switches need to default to false, as they're really awkward to set to false. Ideally, this wouldn't be a double negative.